test/utils/runCodeServerCommand.ts

```diff
@@ -6,9 +6,14 @@ import { promisify } from "util"
  *
  * A helper function for integration tests to run code-server commands.
  */
-export async function runCodeServerCommand(argv: string[]): Promise<{ stdout: string; stderr: string }> {
+export async function runCodeServerCommand(
+  argv: string[],
+  env?: NodeJS.ProcessEnv,
+): Promise<{ stdout: string; stderr: string }> {
   const CODE_SERVER_COMMAND = process.env.CODE_SERVER_PATH || path.resolve("../../release-standalone/bin/code-server")
-  const { stdout, stderr } = await promisify(exec)(`${CODE_SERVER_COMMAND} ${argv.join(" ")}`)
+  const { stdout, stderr } = await promisify(exec)(`${CODE_SERVER_COMMAND} ${argv.join(" ")}`, {
+    env: { ...process.env, ...env },
```

Comment:
Woah I did not know `{ ...undefined }` works.  TIL.

Interestingly `[ ...undefined ]` does not work.  I guess that is JavaScript for you!  :laughing:

Reply:
LOL JS is teaching us new things everyday 😂